chore: Removed Select in Preview (#2796)
Workday/canvas-kit: modules/docs/mdx/12.0-UPGRADE-GUIDE.mdx

<Meta title="Guides/Upgrade Guides/v12.0" />

# Canvas Kit 12.0 Upgrade Guide

This guide contains an overview of the changes in Canvas Kit v12. Please
[reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
any questions.

## Why You Should Upgrade

## Table of contents

- [Removals](#removals)
  - [Select (Preview)](#select-preview)

> 

Reviewer: ```suggestion

## Removals

```

> 

Reviewer: ```suggestion

## Removals

```

> ### Select (Preview)

Reviewer: ```suggestion
### Select (Preview)

**PR:** [#2796](https://github.com/Workday/canvas-kit/pull/2796)
```

> Removed the `Select` component that was in `@workday/canvas-kit-preview-react` and replaced with the `Select` component in `@workday/canvas-kit-react`. Please see the `Select` documentation [here](https://workday.github.io/canvas-kit/?path=/docs/components-inputs-select--basic).

Reviewer: ```suggestion
We've removed the `Select` component that was in Preview. Please use the compound [`Select` component from Main](https://workday.github.io/canvas-kit/?path=/docs/components-inputs-select--basic).
```
